templates/aws/airgap_cluster_nodes/terraform/pools/main.tf

```diff
@@ -0,0 +1,356 @@
+terraform {
+  required_version = ">= 0.13.1"
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~> 3.0"
+    }
+  }
+}
+
+provider "random" {}
+provider "aws" {
+  access_key = var.aws_access_key
+  secret_key = var.aws_secret_key
+  region =  var.aws_region
```

Comment:
Auto format file (I use goland with the [Terraform and HCL plugin](https://plugins.jetbrains.com/plugin/7808-terraform-and-hcl))

Reply:
Oops I forgot to delete this template it is no longer needed.